mbtools/html_to_json.py

```diff
@@ -1,22 +1,48 @@
 import argparse
-import glob
+import os
 import json
 from pathlib import Path
 
 
-def create_json_content(uuid, content, variant="main"):
+def create_json_content(uuid, content):
+
     json_content = {"id": uuid, "content":
-                    [{"variant": variant, "html": content}]}
+                                content}
     return json.dumps(json_content, indent=2)
 
 
 def html_to_json(html_directory, json_directory):
-    for file_name in glob.iglob(f'{html_directory}/*.html'):
-        with open(f'{file_name}') as f:
-            new_file = open(f'{json_directory}/'
-                            f'{Path(file_name).stem}.json', "w")
-            new_file.write(create_json_content(Path(file_name).stem, f.read()))
-            new_file.close()
+
+    for file_name in os.listdir(html_directory):
```

Comment:
This is somewhat opinionated, but given you're using `pathlib.Path`, I'd convert anything you're doing using `os` to `Path` as well. Otherwise this script is needlessly using APIs from across two different modules :man_shrugging: . You can find the equivalent mappings documented [here](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module).